src/scripts/main.js

```diff
@@ -1 +1,70 @@
 'use strict';
+
+document.addEventListener('contextmenu', (e) => {
+  e.preventDefault();
+});
+
+function notification(message, isError = false) {
+  const createNotfication = document.createElement('div');
```

Comment:
There is a typo in the variable name `createNotfication`. It should be `createNotification` to match the intended purpose of the variable.